File: CSharp/DP.Domain/Samples/Facade/MyLogger.cs
using System;
using DP.Domain.Samples.Strategy;

namespace DP.Domain.Samples.Facade
{
    public class MyLogger
    {
        public void Warn(string msg)
        {
             var textLogger = new TextLogger();
             var dbLogger = new DbLogger();
             textLogger.Warn(msg);
             dbLogger.Warn(msg);
        }

        public void Read()
        {
             System.Diagnostics.Trace.WriteLine($"(Database)Dump logs.");
             System.Diagnostics.Trace.WriteLine($"(Text)Dump logs.");
        }
    }
}

File: Python/Samples/Facade/MyLogger.py
import sys
import os.path
sys.path.append(os.path.join(os.path.dirname(__file__), '../Strategy/'))
from DbLogger import DbLogger
from TextLogger import TextLogger

class MyLogger:   
    def warn(self, msg):
        textLogger = TextLogger()
        dbLogger = DbLogger()
        textLogger.warn(msg)
        dbLogger.warn(msg)

    def read(self):
        print("(Database)Dump logs.")
        print("(Text)Dump logs.")
